templates/shelf_jwt_auth/__brick__/{{ project_name.snakeCase() }}/server/lib/router.dart

```diff
@@ -0,0 +1,52 @@
+import 'dart:async';
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:dart_firebase_admin/auth.dart';
+import 'package:shelf/shelf.dart';
+import 'package:shelf_router/shelf_router.dart';
+
+import 'src/firebase.dart';
+import 'src/utils.dart';
+
+Router get router => Router()
+  ..post('/api/auth/register', _signupHandler)
+  ..get('/api/users/me', (req) => checkAuth(req, _getUser));
```

Comment:
does `shelf` support nested pipelines? similar to express `app.get('/some-path', authMiddleware)`

Reply:
Nope, I haven't seen anything like that.